VirtoCommerce.Platform.Web/Controllers/Api/AuthorizationController.cs

```diff
@@ -75,6 +83,18 @@ public async Task<IActionResult> Exchange(OpenIdConnectRequest request)
                 // Create a new authentication ticket.
                 var ticket = await CreateTicketAsync(request, user);
 
+                // Set limited permissions and authenticate user with combined mode Cookies + Bearer.
+                //
+                // LimitedPermissions claims that will be granted to the user by cookies when bearer token authentication is enabled.
+                // This can help to authorize the user for direct(non - AJAX) GET requests to the VC platform API and / or to use some 3rd - party web applications for the VC platform(like Hangfire dashboard).
+                //
+                // If the user identity has claim named "limited_permissions", this attribute should authorize only permissions listed in that claim. Any permissions that are required by this attribute but
+                // not listed in the claim should cause this method to return false. However, if permission limits of user identity are not defined ("limited_permissions" claim is missing),
+                // then no limitations should be applied to the permissions.
+                var claims = await _userClaimsPrincipalFactory.CreateAsync(user);
+                ((ClaimsIdentity)claims.Identity).AddClaim(new Claim(PlatformConstants.Security.Claims.LimitedPermissionsClaimType, _authorizationOptions.LimitedCookiePermissions));
```

Comment:
Please move this logic from there to here.
https://github.com/VirtoCommerce/vc-platform/blob/dev/VirtoCommerce.Platform.Web/ApplicationOAuthProvider.cs#L53